newm-server/src/main/resources/application.conf

```diff
@@ -270,11 +270,12 @@ mintingNotifications {
          messageUrl = "/email/mint-released.html"
          messageUrl = ${?EMAIL_MINT_RELEASED_MESSAGE_URL}
     }
-    passwordChanged {
-         subject = "Your password has been reset /Password reset confirmation"
-         messageUrl = "/email/password-changed.html"
-         messageUrl = ${?EMAIL_PASSWORD_CHANGED_MESSAGE_URL}
-    }
+}
+
+passwordChangeNotification {
+     subject = "Your password has been reset /Password reset confirmation"
```

Comment:
What does the `/` do in the subject line here?

Reply:
Good question. I checked and its actually that way in the copy. Seems weird to me.